src/current/cockroachcloud/export-metrics-standard.md

```diff
@@ -0,0 +1,328 @@
+{{site.data.alerts.end}}
+
+Exporting metrics to AWS CloudWatch is only available on CockroachDB Standard clusters which are hosted on AWS. Metrics export to Datadog is supported on all CockroachDB Standard clusters.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+<a id="the-metricexport-endpoint"></a>
+
+## `metricexport` endpoint
+
+To configure and manage metrics export for your CockroachDB Standard cluster, use the `metricexport` endpoint appropriate for your desired cloud metrics sink:
+
+Cloud metrics sink | Metrics export endpoint
+------------------ | ----------------------------------------------------
+AWS Cloudwatch     | `https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/metricexport/cloudwatch`
+Datadog            | `https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/metricexport/datadog`
+
+Access to the `metricexport` endpoints requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) with the appropriate permissions (`admin` privilege or Cluster Admin role).
+
+The following methods are available for use with the `metricexport` endpoints, and require the listed service account permissions:
+
+Method | Required permissions | Description
+--- | --- | ---
+`GET` | `ADMIN`, `EDIT`, or `READ` | Returns the current status of the metrics export configuration.
+`POST` | `ADMIN` or `EDIT` | Enables metrics export, or updates an existing metrics export configuration.
+`DELETE` | `ADMIN` | Disables metrics export, halting all metrics export to AWS CloudWatch or Datadog.
+
+See [Service accounts]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) for instructions on configuring a service account with these required permissions.
+
+## Enable metrics export
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="aws-metrics-export">AWS CloudWatch</button>
+  <button class="filter-button" data-scope="datadog-metrics-export">Datadog</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-metrics-export">
+
+Exporting metrics to AWS CloudWatch is only available on CockroachDB Standard clusters which are hosted on AWS. If your CockroachDB Standard cluster is hosted on GCP or Azure, you can [export metrics to Datadog](export-metrics.html?filters=datadog-metrics-export) instead.
+
+{{site.data.alerts.callout_info}}
+Enabling metrics export will send around 250 metrics per region to AWS CloudWatch. Review the [AWS CloudWatch documentation](https://aws.amazon.com/cloudwatch/pricing/) to gauge how this adds to your AWS CloudWatch spend.
```

Comment:
For tenants, there will be 84 metrics per region. We could round it up to 90.

Reply:
updated from 250 to 90.